python/MRzeroCore/phantom/sim_data.py

```diff
@@ -126,7 +128,8 @@ def cuda(self) -> SimData:
             self.dephasing_func,
             self.recover_func,
             self.phantom_motion,
-            self.voxel_motion
+            self.voxel_motion,
+            Masks=self.Masks  # no need to put on cuda since it's not use for simulation rather for image metric calculation

```

Comment:
Put it on cuda nevertheless; users might want to calculate the loss on GPU as well (image reconstruction is often done on GPU, not only the simulation)

Reply:
Done,